### ISSUE TITLE
Delete Media?

### ISSUE THREAD

**Issue by sofia-bg** [UNC-Project-Open-AAC/OS-DPI#145]
As I work on a this new layout I am finding that I want different symbols for the same label. I have my spreadsheet automatically name symbols "label + .png" to make my life easier which makes it autopopulate with symbols from old layouts. Also have a few symbols that evidently were corrupted when uploaded, so their names are taken in my OS-DPI media but they do not show a symbol. 

I know I could go through spreadsheet and make unique names for different symbol iterations (play1 play2 etc) but wondering if instead we could offload media. 

Along same vein, could we export media the same way we do spreadsheets? So if someone shares an OS.DPI file and I want to take their media for another project I can? I suppose instead could just save as a new version and offload their spreadsheet so I have the media names already...

**Comment by gbishop** [comment]
I would like to see a design with symbols that you think are corrupted.
That should not happen.

Yes, we need a way to edit media. With the Menus that should be
straightforward. I'll add it to the list.

Exporting media should be possible though it is likely to result in a Zip
file containing the images since triggering multiple downloads would be a
pain...

**Comment by sofia-bg** [comment]
corrupted might be wrong word, they were files from Open Symbols that I exported as .png but were initially .svg and I didn't realize that meant that the actual image wasn't saved even though my computer made a file. See eat example below. 

![Screen Shot 2023-01-09 at 3 01 14 PM](https://user-images.githubusercontent.com/92817124/211397310-447662a9-4ca0-4215-95ea-b55ddce47dd6.png)

**Comment by gbishop** [comment]
If you load a new image with the same name won't it replace it?

**Comment by sofia-bg** [comment]
I thought I tried that, but let me try again

**Comment by aetkie** [comment]
I also wondered about sharing media along with .osdpi files. Would the creator upload the .osdpi file and then a folder with media @gbishop  or upload a zip file with everything contained in one place?

**Comment by gbishop** [comment]
The media are contained within the osdpi file which is actually a Zip file
itself. If you open it, your computer may just recognize that it is a zip
file and unpack it for you.

**Comment by sofia-bg** [comment]
Great! That worked on my mac too

@gbishop @aetkie

**Comment by aetkie** [comment]
This worked for me too! Game changer! Now I don't have to find and save my own symbols. Thank you @gbishop and @sofia-bg
